packages/actor-rdf-metadata-extract-sparql-service/lib/ActorRdfMetadataExtractSparqlService.ts

```diff
@@ -27,28 +28,63 @@ export class ActorRdfMetadataExtractSparqlService extends ActorRdfMetadataExtrac
       // Forward errors
       action.metadata.on('error', reject);
 
-      // Immediately resolve when a SPARQL service endpoint URL has been found
-      const metadata: any = {};
-      action.metadata.on('data', (quad) => {
-        if (quad.predicate.value === 'http://www.w3.org/ns/sparql-service-description#endpoint' &&
-          (quad.subject.termType === 'BlankNode' || quad.subject.value === action.url)) {
-          metadata.sparqlService = quad.object.termType === 'Literal' ?
```

Comment:
This literal check seems to be gone

Reply:
Yes, I figured it would not be needed, if a named node value would also happen to be relative and could be resolved/handled with `resolveIri`? If that is not the case, then I can restore the literal check.

Reply:
Named nodes are always absolute.
Otherwise something would be horribly broken in the parser.

Reply:
This is fixed now!